searchcore/src/vespa/searchcore/proton/server/documentdb.h

```diff
@@ -114,7 +97,6 @@ class DocumentDB : public DocumentDBConfigOwner,
     mutable std::condition_variable _configCV;
     DocumentDBConfig::SP          _activeConfigSnapshot;
     int64_t                       _activeConfigSnapshotGeneration;
-    SerialNum                     _activeConfigSnapshotSerialNum;
```

Comment:
Why is this removed?

Reply:
It was not used. It was set, but never used. I think it leftovers from when @toregge rewrote how we do TLS replay.